grantnav/frontend/templates/search.html

```diff
@@ -338,6 +356,35 @@ <h1>Search input is not valid</h1>
 </div>
 {% endblock %}
 
+{% block models %}
+    <div class="modal fade" id="summary-info-model" tabindex="-1" role="dialog" aria-labelledby="myModalLabel">
+      <div class="modal-dialog modal-sm" role="document">
+        <div class="modal-content">
+          <div class="modal-header">
+            <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+            <h4 class="modal-title" id="myModalLabel">Award amounts in other currencies</h4>
+          </div>
+          <div class="modal-body">
+            {% for bucket in results.aggregations.currency_stats.buckets %}
+              {% if not forloop.first %}
+                {% with currency=bucket|get:"key"|upper %}
+                  <dl class="dl-horizontal top-stats-search">
+                    <dt>Total {{currency}} grants</dt> <dd>{{bucket|get:"amount_stats"|get:"count"|get_amount}}</dd>
```

Comment:
Given that ```get``` filter defaults to ```None```, I think this chains of ```get```s might _in principle_ give a 500 (_" 'NoneType' object has no attribute 'get' "_)